backend/apps/sandbox/views.py

```
@@ -235,12 +313,104 @@ def get_queryset(self):
 
 
 class CodeSnippetViewSet(viewsets.ModelViewSet):
+    """
+    ViewSet for CodeSnippet model with filtering.
+    """
     serializer_class = CodeSnippetSerializer
     permission_classes = [permissions.IsAuthenticated]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)